### ISSUE TITLE
nsubj or csubj for clausal subject of стоит

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Russian-SynTagRus#9]
I'm looking at various constructions in Russian, one of them is "NP-Dat стоит VP-Inf". The structure I have been using is: `[root [obl NP-Dat] стоит [csubj VP-Inf]]`, but it seems that these are annotated with `nsubj` in SynTagRus. Should they be `csubj` ?

```
# source = 2003Nelegalnaya_perepis.xml 31
# text = Но стоит закрыть лишь глаза, отвлечься на мгновенье, как он распнет Христа, лишит людей воды из-под крана, уронит на Родину метеорит, подогреет наших младенцев в родильных домах и выпьет их кровь, оберет целые народы, приведет в действие тайные механизмы Сиона, делающие людей рабами, отравит колодцы, развяжет войну, накличет псориаз?
1       Но      но      CCONJ   _       _       2       cc      2:cc    _
2       стоит   стоить  VERB    _       Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act      0       root    0:root  _
3       закрыть закрывать       VERB    _       Aspect=Perf|VerbForm=Inf|Voice=Act      2       nsubj   2:nsubj _
4       лишь    лишь    PART    _       _       5       advmod  5:advmod        _
5       глаза   глаз    NOUN    _       Animacy=Inan|Case=Acc|Gender=Masc|Number=Plur   3       obj     3:obj   SpaceAfter=No
6       ,       ,       PUNCT   _       _       5       punct   5:punct _
```

**Comment by jnivre** [comment]
Is this something like dative subjects with psych verbs? We need to discuss these (and other quirky case constructions) in a central group, I think.

**Comment by dan-zeman** [comment]
No, the question is not whether subject or non-subject. The question is why the subject is nsubj and not csubj. My preference is to use csubj when it is headed by infinitive. I believe that this is what we do in Czech.

**Comment by jnivre** [comment]
Okay, sorry.

**Comment by Kira-D** [comment]
scubj was only implemented in 2.1
